src/server/svr_func.c

```diff
@@ -7166,10 +7177,14 @@ unlicense_socket_licensed_nodes(void)
 int
 set_sched_throughput_mode(attribute *pattr, void *pobj, int actmode)
 {
-	if (actmode == ATR_ACTION_ALTER || actmode == ATR_ACTION_RECOV) {
-		if (pbs_conf.pbs_use_tcp == 0) {
-			return PBSE_BADATVAL;
+	if (  pobj == dflt_scheduler ) {
```

Comment:
Scope of the PR is to  address only creating/modify/deleting sched objects.  There are some changes made to accommodate the default scheduler behavior. You will see full implementation when other interfaces are implemented.